### PR TITLE
Only query single route if segmentRange specified

### PR DESCRIPTION
**Pull request by sshane**
Removes two queries on loading a route on master and the other attempt at this

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -148,7 +166,12 @@ export function urlForState(dongleId, log_id, start, end, prime) {
 }
 
 function updateTimeline(state, dispatch, log_id, start, end, allowPathChange) {
-  dispatch(checkRoutesData());
```
Comment (sshane) [inline src/actions/index.js:151]: Added in https://github.com/commaai/connect/pull/419, I'm not sure why. It re-queries the API twice on loading any route (from URL only). With this change, this ends up breaking something in `ACTION_ROUTES_METADATA` as it is also called twice; the second time `segmentRange`'s `start` and `end` values are properly set, but that function only works if they're both `NaN` on first load (??). On master, querying the API the first time here is fine because the `limit` param just happens to be set to 0 so `currentRoute` isn't set yet, which would change `segmentRange`'s `start` and `end` lol

In summary: I'm amazed connect works as well as it does

**Comment by sshane** [comment]
Implemented these changes in: https://github.com/commaai/connect/pull/522
